### DISCUSSION TITLE
FastDDS n docker

### DISCUSSION THREAD

**Discussion by iekhanx** [eProsima/Fast-DDS#2637] [Support]
Hi there,
I managed to install the FastDDS on the ubuntu 22.04 server and build the examples.
I also installed the Docker service on the server.
several remote user accounts have been created on the server ready to use.
NOW how one could use the server remotely? 
Can you guide, please?
thank you,

**Comment by JLBuenoLopez** [comment]
Sorry @iekhanx,

I am not understanding what are you trying to do. Could you give me more insight into what is your purpose?
